### PR TITLE
0.13: implement base router validateActionOutputs tests

### PR DESCRIPTION
**Pull request by Orzelius**

### CODE REVIEW

File: core/test/unit/src/router/base.ts [hunk 1]
```diff
@@ -431,20 +438,57 @@ describe("BaseActionRouter", () => {
...
-      throw "TODO"
+      const { router } = await createTestRouter(testPlugins)
+
+      await expectError(async () => await router.validateActionOutputs(resolvedBuildAction, "runtime", { foo: 123 }), {
+        contains: "Error validating runtime action outputs from Build 'module-a': key .foo must be a string.",
+      })
     })
 
     it("throws if no schema is set and a key is set", async () => {
       throw "TODO"
```
Comment (Orzelius) [inline core/test/unit/src/router/base.ts:471]: Currently [validateActionOutputs](https://github.com/garden-io/garden/blob/0.13/core/src/router/base.ts#L317) doesn't throw if the schema is not defined and an output is passed. Which is the intended behavior?

Comment (Walther) [reply core/test/unit/src/router/base.ts:471]: No idea - need to ask someone with more context knowledge 😅

Comment (edvald) [reply core/test/unit/src/router/base.ts:471]: I believe that is an actual issue, but it's not really a major concern, you can just skip the test.

**Comment by edvald** [comment]
I wonder what's going on with the `test-framework` step here... Those tests are certainly not _actually_ passing.

**Pull request merged**